src/librustc_typeck/check/method/probe.rs

```diff
@@ -1218,8 +1205,10 @@ impl<'a, 'gcx, 'tcx> ProbeContext<'a, 'gcx, 'tcx> {
         // are given do not include type/lifetime parameters for the
         // method yet. So create fresh variables here for those too,
         // if there are any.
-        assert_eq!(substs.types().count(), method.generics.parent_types as usize);
-        assert_eq!(substs.regions().count(), method.generics.parent_regions as usize);
+        assert_eq!(substs.types().count(),
```

Comment:
why did rustfmt convert this? These expressions are happily fit on a  single line.